docs: additional bubbles (#583)
charmbracelet/bubbles: README.md

}
```

## Additional Bubbles

There are also
[community-owned bubbles.](https://github.com/charm-community/additional-bubbles)

> Check out its README for instructions on how to get your bubble added to this list!

Reviewer: ```suggestion
Check out the project’s README for instructions on how to get your bubble added to the list!
```

> We have a list of
> [community-created bubbles](https://github.com/charm-community/additional-bubbles)
> in our [charm-community](https://github.com/charm-community) GitHub organization.

Reviewer 2: ```suggestion
There are also
[community-owned bubbles.](https://github.com/charm-community/additional-bubbles)
Check out its README for instructions on how to get your bubble added to this list!
```
